core/src/test/scala/org/apache/spark/storage/ShuffleBlockFetcherIteratorSuite.scala

```diff
@@ -392,7 +391,7 @@ class ShuffleBlockFetcherIteratorSuite extends SparkFunSuite with PrivateMethodT
 
     configureMockTransfer(Map())
     val iterator = createShuffleBlockIteratorWithDefaults(
-      Map(hostLocalBmId -> toBlockList(hostLocalBlocks.keys, 1L, 1))
+      Map(blockManager.blockManagerId -> toBlockList(hostLocalBlocks.keys, 1L, 1))
```

Comment:
Wounldn't it be treated as local blocks rather than host-local blocks?

Reply:
Fixed the test properly... Do you mind taking a look again?